src/Backends/Consul/ConsulBackend.cxx

```diff
@@ -58,12 +88,41 @@ boost::optional<std::string> ConsulBackend::getString(const std::string& path)
 
 boost::property_tree::ptree ConsulBackend::getRecursive(const std::string& path)
 {
-  auto requestKey = replaceDefaultWithSlash(addPrefix(path));
+  auto requestKey = replaceDefaultWithSlash(addPrefix(path)) + "[]";
```

Comment:
Is `[]` appended to every request?

Reply:
This is the only way to have one method (`getRecursive`) providing trees and arrays. To get rid of it I could append interface with `getArray`.